Userland/Libraries/LibJS/AST.h

```diff
@@ -169,7 +169,7 @@ class LabelledStatement : public Statement {
     virtual Completion execute(Interpreter&) const override;
```

Comment:
Commit message: you should probably be a bit more specific about where you replaced `DeprecatedString` since it is definitely not all occurences in LibJS that you took care of :^)